src/components/SessionPromptModal/SessionPromptModal.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import CTAButton from '../CTAButton/CTAButton';
+
+interface SessionPromptModalProps {
+  continueSession: () => void;
+  setShowSessionPrompt: (showSessionPrompt: boolean) => void;
+  showSessionPrompt: boolean;
+}
+
+export default function SessionPromptModal({
+  continueSession,
+  setShowSessionPrompt,
+  showSessionPrompt,
+}: SessionPromptModalProps) {
+  if (showSessionPrompt)
+    return (
+      <div className='cio-session-prompt-modal'>
```

Comment:
How hard would it be to add a click outside event that does the same thing as no?

Reply:
Added